### ISSUE TITLE
Bundled outdated ffprobe binary cannot properly recognize some media files

### ISSUE THREAD

**Issue by zestysoft** [Radarr/Radarr#9764]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Using this container: linuxserver/radarr:nightly-version-5.3.5.8579

I am getting a number of `DetectSample: Failed to get runtime from the file, make sure ffprobe is available errors in the logs.` errors.
Digging deeper shows that ffprobe is failing with "Invalid argument" for all.
Example:
```
[v5.3.5.8579] FFMpegCore.Exceptions.FFMpegException: ffprobe exited with non-zero exit-code (1 - /path/file.mkv: Invalid argument)
 at FFMpegCore.FFProbe.GetStreamJson(String filePath, Int32 outputCapacity, FFOptions ffOptions)
 at NzbDrone.Core.MediaFiles.MediaInfo.VideoFileInfoReader.GetMediaInfo(String filename) in ./Radarr.Core/MediaFiles/MediaInfo/VideoFileInfoReader.cs:line 140
```
If I shell into the container and directly run: /app/radarr/bin/ffprobe /path/file.mkv I see the same error:
```
# /app/radarr/bin/ffprobe file.mkv
ffprobe version 5.1.4 Copyright (c) 2007-2023 the FFmpeg developers
  built with gcc 9 (Ubuntu 9.4.0-1ubuntu1~20.04.2)
  configuration: --prefix=/ffmpeg/build --bindir=/ffmpeg/bin --logfile=/ffmpeg/bin/log.txt --disable-autodetect --disable-iconv --enable-libdav1d --disable-programs --enable-ffprobe --disable-avdevice --disable-swresample --disable-swscale --disable-postproc --disable-avfilter --disable-network --disable-everything --enable-protocol=file --enable-demuxers --enable-parsers --enable-decoders --enable-bsf=av1_frame_split --enable-bsf=av1_frame_merge --enable-bsf=av1_metadata --pkg-config-flags=--static --extra-ldexeflags=-static --extra-libs='-lpthread -lm'
  libavutil      57. 28.100 / 57. 28.100
  libavcodec     59. 37.100 / 59. 37.100
  libavformat    59. 27.100 / 59. 27.100
file.mkv: Invalid argument
```
ffprobe is showing version 5.1.4
If I run the same command from my computer using the 6.1.1 version against the same mkv file, it works as expected.

Any chance it's possible to bump the binary that's getting installed in the container? From what I can tell, it  looks like the container's files are getting populated from a tar file hosted by y'all:
https://github.com/linuxserver/docker-radarr/blob/a95508d1d21be34f2760a4a7b91dd3e87e4d58eb/Dockerfile#L30

https://radarr.servarr.com/v1/update/nightly/updatefile?version=5.3.5.8592&os=linuxmusl&runtime=netcore&arch=x64

I attempted a PR with the docker-radarr repo, but was shot down and told to come here:
https://github.com/linuxserver/docker-radarr/pull/220

It's not clear to me how I can otherwise provide a PR for the tar blob that the Dockerfiles use.

### Expected Behavior

ffprobe properly probes the media file.

### Environment

```markdown
- OS: linux
- Radarr: 5.3.5.8579
- Docker Install: Using container image?  Yes, in k8s.
- Using Reverse Proxy: N/A
- Browser: N/A
- Database: N/A
```

### What branch are you running?

Nightly

### Trace Logs? **Not Optional**

[v5.3.5.8579] FFMpegCore.Exceptions.FFMpegException: ffprobe exited with non-zero exit-code (1 - /path/file.mkv: Invalid argument)
 at FFMpegCore.FFProbe.GetStreamJson(String filePath, Int32 outputCapacity, FFOptions ffOptions)
 at NzbDrone.Core.MediaFiles.MediaInfo.VideoFileInfoReader.GetMediaInfo(String filename) in ./Radarr.Core/MediaFiles/MediaInfo/VideoFileInfoReader.cs:line 140

### Trace Logs have been provided as applicable. Reports will be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by bakerboy448** [comment]
The build process pulls the custom build of ffprobe. It's not as simple as a version bump

- https://github.com/Servarr/FFmpeg
- https://github.com/Radarr/Radarr/blob/5f624a147bb62d37b731d9a0ae02bfd338793962/src/NzbDrone.Core/Radarr.Core.csproj#L12-L13

**Comment by Qstick** [comment]
5.1 is LTS and still receives fixes as appropriate (5.1.4 was just released a few months ago). I've not seen many MKV if any that didn't get read properly with 5.1, but it would be beneficial to see the output of the command that @mynameisbogdan mentioned. 

Do you have further details on this file/release so that we may test? 

> Digging deeper shows that ffprobe is failing with "Invalid argument" for all.

If it's every file, then it's something else going on.

**Comment by zestysoft** [comment]
Forgive me.  For whatever reason I'm not able to reproduce this with the same files I originally had trouble with so I don't want to cause any undue work.  I'm not sure if the issue was transitory and restarting the container with the newer binary happened to be a coincidence, or if some type of sanitization happened when the files moved from the download client location to the final destination.

It's just weird because I did restart the container a few times without upgrading the binary and the problem seemed to be persistent until I upgraded the binary, but that did take time for me to figure out how to replace and maybe the problem fixed itself in the meantime.

If you don't mind, please leave this open for a bit as I've got a number of other files down the pike.  I've been focusing on 3D content and this happened to 12 different files so far.  Mind you these were "completed" files where the seeding was finished, and I didn't have trouble with the files outside of ffprobe.

**Comment by zestysoft** [comment]
I can confirm the problem I'm dealing with is not due to the version of the ffprobe binary.  Sorry for the trouble!

**Comment by zestysoft** [comment]
I'm adding some additional details here in case google drops people into this thread.

The radarr container running in a pod in a self-hosted kubernetes cluster uses a samba mount via a persistent volume / claim via the smb.csi.k8s.io driver.
For whatever reason, likely an overtaxed NAS, the samba connection produced an error (seen via dmesg):
```
[ 8820.139215] CIFS: VFS: \\server sends on sock 000000002e6dae34 stuck for 15 seconds
[ 8820.139239] CIFS: VFS: \\server Error -11 sending data on socket to server
[ 8822.443314] CIFS: VFS: Send error in read = -11
[ 9442.572828] CIFS: Attempting to mount \\server\data
[ 9442.577064] CIFS: VFS: parse_server_interfaces: malformed interface info
```
which apparently put the mount into an inconsistent state -- directory traversal still worked, but access to the files failed.
I was able to get that "invalid argument" directly on the node and it didn't matter which binary I attempted to use. I guess that error message is generic?

The fix, and probably what happened originally when I replaced the ffprobe binary here thinking it fixed my issue, was to move the pod to a different node which would have a different mount to the samba server.
